examples/user_guide/Interactive.ipynb

```diff
@@ -327,15 +325,21 @@
    "outputs": [],
    "source": [
     "diff = ds.air.interactive.sel(time=pnw.DiscreteSlider) - ds.air.mean('time')\n",
-    "kind = pnw.Select(options=['contourf', 'contour', 'image'])\n",
-    "layout = diff.hvplot(cmap='RdBu_r', clim=(-20, 20), kind=kind).layout\n",
+    "kind = pnw.Select(options=['contourf', 'contour', 'image'], value='image')\n",
+    "interactive = diff.hvplot(cmap='RdBu_r', clim=(-20, 20), kind=kind)\n",
     "\n",
-    "pn.Column(pn.Row(pn.panel(\"https://hvplot.holoviz.org/assets/hvplot-wm.png\", width=100), \n",
-    "                 pn.Spacer(width=20),\n",
-    "                 pn.Column(pn.panel(\"## Select a time and type of plot\", width=400),\n",
-    "                           layout[0][0]),\n",
-    "                 pn.panel(\"https://panel.holoviz.org/_static/logo_stacked.png\", width=100)),\n",
-    "          layout[0][1]).servable()"
+    "pn.Column(\n",
+    "    pn.Row(\n",
+    "        pn.panel(\"https://hvplot.holoviz.org/assets/hvplot-wm.png\", width=100), \n",
+    "        pn.Spacer(width=20),\n",
+    "        pn.Column(\n",
+    "            pn.panel(\"## Select a time and type of plot\", width=400),\n",
+    "            interactive.widgets\n",
+    "        ),\n",
+    "        pn.panel(\"https://panel.holoviz.org/_static/logo_stacked.png\", width=100)\n",
+    "    ),\n",
+    "    interactive.panel()\n",
```

Comment:
Is there a good reason why interactive.panel() has method-like syntax while interactive.widgets has attribute-like access? If so, fine, but it does seem confusing and hard to remember.

Reply:
Good point, I'll standardize on method syntax.